turbo/execution/eth1/metrics.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2024 The Erigon Authors
+// This file is part of Erigon.
+//
+// Erigon is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Erigon is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with Erigon. If not, see <http://www.gnu.org/licenses/>.
+
+package eth1
+
+import (
+	"time"
+
+	"github.com/erigontech/erigon-lib/metrics"
+)
+
+var (
+	updateForkChoiceArrivalDelay = metrics.NewSummary(`update_fork_choice_arrival_delay`)
+	updateForkChoiceDuration     = metrics.NewSummary(`update_fork_choice_duration`)
+	updateForkChoiceDepth        = metrics.NewGauge(`update_fork_choice_fork_depth`)
```

Comment:
Wouldn't it be more useful to have this as a summary too? Making it a gauge means we will only have the depth of the latest fork choice and we won't be able to see any reorgs in time.